phpunit.xsd

```diff
@@ -171,6 +171,12 @@
       <xs:enumeration value="random"/>
     </xs:restriction>
   </xs:simpleType>
+  <xs:simpleType name="executionOrderDefectsType">
```

Comment:
this shall be removed now, and `executionOrderType` shall be extended to contain `defects` and `depends`

Reply:
Removed. I have added a list of the common scenarios that make sense. Couldn't find a nice way to define the list in the XSD to help the tooling with validation and autocomplete.